src/main/java/seedu/momentum/logic/commands/FindCommand.java

```diff
@@ -14,14 +20,22 @@
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all projects whose names contain any of "
+    public static final String FILTER_ALL_MATCH = "all";
```

Comment:
I think we should standardise to FIND_ALL_MATCH and FIND_ANY_MATCH for uniformity since filter might be confusing/misleading